### PR TITLE
Altevian Adjustments - Descriptions & Small Tweaks

### PR DESCRIPTION
**Pull request by Enzo-Leon**
Adjusts a lot of the wording on the existing altevian items in game. Shortens stuff while improving the flow of the descriptions.

Also a few little tweaks to their weapons and generator.

Otherwise, this is mostly just flavor and lore adjustments.

### CODE REVIEW

File: code/modules/clothing/under/accessories/altevian_vr.dm [hunk 1]
```diff
@@ -81,13 +81,13 @@
...
 	name = "Altevian Civilian Badge"
-	desc = "An emblem commonly seen worn by the altevians off-work or by visitors of their ships."
+	desc = "An emblem that's normally worn by non-altevian workers on their fleet ships."
 	icon_state = "altevian_badge"
 	slot = ACCESSORY_SLOT_MEDAL
 
 /obj/item/clothing/accessory/altevian_badge/aquila
 	name = "Royal Altevian Navy Emblem"
-	desc = "An emblem commonly seen worn by the altevians for their work operations."
+	desc = "An emblem found across all altevian vessels. The unique metals and jewels denote ranks for them back home. Otherwise, it's just a pretty rat skull emblem with a set of crystals for their eyes and fangs."
```
Comment (KillianKirilenko) [inline code/modules/clothing/under/accessories/altevian_vr.dm:90]: see earlier comment re: unique/specific

File: code/game/objects/structures/props/altevian.dm [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /obj/structure/prop/altevian_generator_wrecked
-	name = "Converted High Energy Exchange Supplier Extractor"
-	desc = "A take on the classic PACMAN reactors that are seen throughout the galaxy. The altevians have ripped apart the tech and seemed to of found a way to maximize the fuel usage one would see with this kind of process. \
-			However, it is a lot bulkier and nearly impossible to break apart, but still can be moved if need be with special tools. This one appears to be totally wrecked though."
+	name = "Phoronic Conversion System"
+	desc = "A reactor system similar to the PACMAN generators seen throughout the stars. This one is a specific model created by the altevians. It seems this reactor has a way to maximize the fuel usage one would see with this kind of process. \
```
[the remark this answers is not in the document]
Comment (Enzo-Leon) [reply code/game/objects/structures/props/altevian.dm:3]: Adjusted!

File: code/modules/clothing/under/accessories/altevian_vr.dm [hunk 3]
```diff
@@ -46,7 +46,7 @@
 
 /obj/item/clothing/accessory/scarf/altevian
 	name = "Blue Comfortable Scarf"
-	desc = "A scarf made of synthetic material that would be common among a people who don't really have the leisure of organic crops for luxury goods. They seemed to of refined this process though and one wouldn't be able to tell the difference between it and ones made normally."
+	desc = "A soft silk like scarf made of synthetic material. It's soft to the touch and feels like actual silk."
```
[the remark this answers is not in the document]
Comment (Enzo-Leon) [reply code/modules/clothing/under/accessories/altevian_vr.dm:49]: Adjusted, was unsure myself.

File: code/game/objects/items/weapons/melee/energy_vr.dm [hunk 4]
```diff
@@ -9,7 +9,7 @@
 
 /obj/item/weapon/melee/energy/sword/altevian
 	name = "plasma blade cutter"
-	desc = "A device used as both defense and operational purposes to cut through most metals. This is usually seen on engineers from the altevian hegemony when working salvaging derelicts."
+	desc = "A device that's seen use as both a defense, and standard cutter to melt through most metals. It's normally seen on engineers from the Altevian Hegemony when salvaging derelicts."
```
[the remark this answers is not in the document]
Comment (Enzo-Leon) [reply code/game/objects/items/weapons/melee/energy_vr.dm:12]: Adjusted

File: code/modules/client/preference_setup/loadout/loadout_xeno_vr.dm [hunk 5]
```diff
@@ -105,7 +105,7 @@
 	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(pants))
 
 /datum/gear/accessory/altevian_aquila
-	description = "An emblem commonly seen worn by the altevians for their work operations."
+	description = "An emblem found across all altevian vessels. The unique metals and jewels denote ranks for them back home. Otherwise, it's just a pretty rat skull emblem with a set of crystals for their eyes and fangs."
```
[the remark this answers is not in the document]
Comment (Enzo-Leon) [reply code/modules/client/preference_setup/loadout/loadout_xeno_vr.dm:108]: Adjusted!

File: code/modules/clothing/suits/miscellaneous_vr.dm [hunk 6]
```diff
@@ -151,7 +151,7 @@
 // Altevian admiralty stuff
 /obj/item/clothing/suit/captunic/capjacket/altevian_admiral				// Subtype of capjacket because A) it makes sense and B) conviniently matching stats
 	name = "altevian officer's suit"
-	desc = "Formal attire worn by officers and bridge crew from the Altevian Hegemony. The material is made of high quality silk and provides maximum comfort and breathing room for those that are working double shifts all the time."
+	desc = "A formal jacket worn by the bridge and command crew from the Altevian Hegemony. The material is of high quality silk, and provides maximum comfort and breathing room for those that are working for hours on end all the time."
```
[the remark this answers is not in the document]
Comment (Enzo-Leon) [reply code/modules/clothing/suits/miscellaneous_vr.dm:154]: Removed the ending statements.

File: code/modules/clothing/under/accessories/altevian_vr.dm (hunk 1 above)
Comment (Enzo-Leon) [reply code/modules/clothing/under/accessories/altevian_vr.dm:90]: And adjusted!

**Pull request merged**